build/docker/dos-service-fuzzy-search-api/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM 461183108257.dkr.ecr.eu-west-2.amazonaws.com/uec/service-finder/api-base:1.0.20191119
```

Comment:
Can we base your image on this template `build/automation/lib/docker/template/openjdk/Dockerfile`? Basically most (not everything) from the `build/automation/lib/docker/template/openjdk` directory to `build/docker/dos-service-fuzzy-search-api`

Reply:
Yes, done.